### PR TITLE
[FT-BCN-012020]-Tarek.Koushaji

### PR DESCRIPTION
**Pull request by Tking1606**
Clue Lab is done

### CODE REVIEW

File: starter-code/src/clue.js [hunk 1]
```diff
@@ -92,5 +92,23 @@ const weaponsArray = [
...
 // ITERATION 2
+function selectRandom(arr){
+ return arr[Math.floor(Math.random() * arr.length)];
 
+}
+function pickMystery(){
+    
+    
+    var x={
+        suspect :selectRandom.suspectsArray,
```
Comment (tawebbcn) [inline starter-code/src/clue.js]: This returns undefined. To get a random item it should have been something like suspect:selectRandom(suspectsArray)

File: starter-code/src/clue.js [hunk 2]
```diff
@@ -92,5 +92,23 @@ const weaponsArray = [
...
+        room : selectRandom.roomsArray,
+        weapon : selectRandom.weaponsArray
+    }
+    return x;
+}
 // ITERATION 3
+function revealMystery(obj){
```
Comment (tawebbcn) [inline starter-code/src/clue.js]: this also returns undefined because of the prevous issue

File: starter-code/src/clue.js [hunk 3]
```diff
@@ -1,13 +1,95 @@
...
 // Suspects Collection
-const suspectsArray = [];
+const suspectsArray = [
+    {
+     firstName: 'Jacob',
+     lastName: 'Green',
+     occupation: 'Entrepreneur',
+     age: 45,
+     description: 'He has a lot of connections',
+     image: mrGreenImage.src='//pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg',
```
Comment (tawebbcn) [inline starter-code/src/clue.js]: this throws an error "mrgreenImage is not defined". I think you might have confused the image property of this particular object with an HTML tag. It was supposed to be:

image: '//pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg',

**Comment by tawebbcn** [comment]
good job overall. You had some mistakes that will improve with a little more practice but you're doing very well, Tarek. Keep it up! :)

File: starter-code/src/clue.js (hunk 1 above)
Comment (Tking1606) [reply starter-code/src/clue.js]: I tried this and it was still undefined ....

File: starter-code/src/clue.js (hunk 3 above)
Comment (Tking1606) [reply starter-code/src/clue.js]: that was a mistake ... I wasn't that focused :)

File: starter-code/src/clue.js (hunk 2 above)
Comment (Tking1606) [reply starter-code/src/clue.js]: > good job overall. You had some mistakes that will improve with a little more practice but you're doing very well, Tarek. Keep it up! :)

really thanks ..... these comments cheers me up every time I read them XD

**Pull request closed without merging**